src/main/scala/fly/play/jiraExceptionProcessor/Jira.scala

```diff
@@ -46,24 +45,30 @@ class Jira(
   }
 
   private def findIssue(hash: String, componentId: String, hashCustomField: String) =
-    playProjectIssueFormat flatMap { implicit format =>
+    playProjectIssueFormat flatMap { implicit format => {
+      val searchOnHash = configuration.hashCustomFieldType match {
+        case "UUID" => s"${configuration.hashCustomFieldName} = $hash "
```

Comment:
These `"UUID"` and `"TEXT"` strings should be placed in some value in some object